### DISCUSSION TITLE
Hue-compatible LED light "unsupported" ? Generic driver?

### DISCUSSION THREAD

**Discussion by ManfredBartz** [Koenkk/zigbee2mqtt#10910] [Devices]
I have an ALCHEMY_VIII LED down-light from "Apollo Lighting".  The manufacturer claims Hue-compatibility.
On Hubitat Elevation I can fully control the device by assigning a generic driver to control color, brightness and ramp-speed.
Is there an equivalent functionality in Z2M to assign an existing driver to this device?
I know that one can write a converter but that seems like a lot of unnecessary work when the required functionality very likely already exists in Z2M.  Also, at this time I am still trying to figure out how to extract the necessary data from the device.

**Comment by Koenkk** [comment]
If you fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+ this device can be supported
